gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -6,12 +6,16 @@ SCL_PREFIXES = JSON.load(File.read(File.join(`git rev-parse --show-toplevel`.str
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
+%{!?_root_datadir:%global _root_datadir %{_datadir}}
+%{!?_root_localstatedir:%global _root_localstatedir %{_localstatedir}}
+%{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
 %global foreman_proxy_min_version 1.24
-%{!?_root_datadir:%global _root_datadir %{_datadir}}
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
+%global foreman_proxy_statedir %{_root_localstatedir}/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
 %global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
```

Comment:
Note that this is actually incorrect and I didn't catch it in 19ada47719ee878439561a5e40027c8bcb5cae0f. I think the correct version is:
```suggestion
%global smart_proxy_dynflow_bundlerd_dir %{?rhel:/opt/theforeman/tfm/root}%{_root_datadir}/smart_proxy_dynflow_core/bundler.d
```